src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/JsonFileDeserializer.cs

```diff
@@ -16,7 +16,7 @@ private class DefaultJsonFileDeserializer : JsonFileDeserializer
         {
             public override TValue Deserialize<TValue>(string filePath) where TValue : class
             {
-                using var stream = new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.ReadWrite | FileShare.Delete);
+                using var stream = new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.Read);
```

Comment:
I tried out this change to make sure it works on top of my change but I see,
![image](https://user-images.githubusercontent.com/1579269/78726782-b8b6fd00-78e7-11ea-820f-cdc2e2a22614.png)

I don't think my changes have anything to do with this. Are there other changes you did that you didn't include in this PR?

Reply:
Ohhhh ya I recall this now. That's why I had added that ReadWrite / Delete flag.

Reply:
If I disable `IOException`, things seem to work fine in spite of this though. @ryanbrandenburg do you have it disabled accidentally? That could be why you weren't seeing it.

Reply:
IIRC exceptions like these end up crashing VSCode's language server I believe

Reply:
> If I disable IOException, things seem to work fine in spite of this though

I take that back. It doesn't crash but I see a bunch of inconsistencies with completions.